rleplus/internal/bitvector_test.go

```diff
@@ -0,0 +1,125 @@
+package bitvector_test
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/filecoin-project/go-filecoin/rleplus/internal"
+	tf "github.com/filecoin-project/go-filecoin/testhelpers/testflags"
+	"gotest.tools/assert"
+)
+
+// Note: When using this helper assertion, expectedBits should *only* be 0s and 1s.
+func assertBitVector(t *testing.T, expectedBits []byte, actual bitvector.BitVector) {
+	assert.Equal(t, len(expectedBits), actual.Len)
+
+	for idx, bit := range expectedBits {
+		actualBit, err := actual.Get(idx)
+		assert.NilError(t, err)
+		assert.Equal(t, bit, actualBit)
+	}
+}
+
+func TestBitVector(t *testing.T) {
+	tf.UnitTest(t)
+
+	t.Run("Zero Value", func(t *testing.T) {
```

Comment:
Nit: there are some inconsistencies, but we mostly use lower case for test run names.